### ISSUE TITLE
NVDA stops reading when encountering inline links or bold text (mouse hover)  :

### ISSUE THREAD

**Issue by sharpmz-kh** [nvaccess/nvda#18504]
System information:

    Operating system: Windows 10

    Input: Mouse

    Screen reader: NVDA only, no others

    NVDA version: 2025.1.2 (Build 2025.1.2.36913)

    Firefox version: 140.0.4 (64-bit)

    Microsoft Edge version: 138.0.3351.83 (official build, 64-bit)

Issue description:

I have a reproducible problem with NVDA (version 2025.1.2.36913) on Windows 10:

When visiting any website that contains bold text or linked text, NVDA does not read the full line continuously.

Example page to reproduce the issue:
https://github.com/nvaccess/nvda/issues/18492
Steps to reproduce:

    Hover the mouse over the beginning of this sentence on the page:
    “Please reproduce this issue and provide a log file of the behaviour. Ensure your log level is set to debug in general preferences.”

    NVDA starts reading, but stops at the link text (“log file”).

    Hovering directly on the link text, NVDA reads only the link.

    Hovering the mouse after the link, NVDA resumes reading the rest of the sentence.

This behavior occurs with:

    All browsers tested (Firefox and Edge)

    All websites that include <a>, <strong> or <b> tags

    It also happens in Word and emails in Thunderbird

I can reproduce this behavior consistently. It seems to be a general issue with inline formatting elements like links or bold text.

Unfortunately, I currently don’t know how to create an NVDA log file. If needed, I will gladly provide one later.

Thank you very much for your support.

**Comment by seanbudd** [comment]
Welcome @sharpmz-kh 

We are unable to process this issue as it stands.

- Please fill out the issue template fully.
- Please reproduce this issue and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences.

Kind Regards,
NV Access Software Developers

**Comment by sharpmz-kh** [comment]
I would like to send you whatever you  want, but it would very helpful to inform how to get the information, thanks.

**Comment by seanbudd** [comment]
@sharpmz-kh
- as linked above, you can follow our wiki to get a log file: https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps
- we also have this document explaining how to fill out the bug report template: https://github.com/nvaccess/nvda/blob/master/projectDocs/issues/githubIssueTemplateExplanationAndExamples.md#bug-report-template
- This is the template you need to fill out: https://github.com/nvaccess/nvda/issues/new?template=bug_report.md

Once you've read through above feel free to ask any follow up questions you have
